server/src/routes/appointments.routes.js

```diff
@@ -25,7 +25,7 @@ router.get("/:email/all", getUsersAppointments);
 // PATCH "/appointments/cancel"
 router.patch("/cancel", cancelAppointment);
 
-// PATCH "/appointments/:id"
-router.patch("/:id", updateVisited);
+// PATCH "/appointments/:id/status-change"
+router.patch("/:id/status-change", updateStatus);
```

Comment:
```suggestion
router.patch("/:id/status", updateStatus);
```
FYI I think this would be more standard semantically, since the verb PATCH carries the meaning of it being an update already

Reply:
gotcha